exec.go

```diff
@@ -142,7 +142,7 @@ func (db *DB) InsertBulk(table string, data interface{}) error {
 	}
 
 	if structMode {
-		return fmt.Errorf("InsertBulk: Need Slice to insert bulk.")
+		return fmt.Errorf("need slice to insert bulk")
```

Comment:
I prefer starting with the function name, so "InsertBulk: need slice".

Reply:
It's a refactor with that practice (don't start with uppercase and end with punctuation).
Also, You mentioned the function name in the sentence: need slice to **insert bulk**